crates/notedeck_ui/src/note/mod.rs

```diff
@@ -782,6 +782,7 @@ fn render_note_actionbar(
     i18n: &mut Localization,
 ) -> egui::InnerResponse<Option<NoteAction>> {
     ui.horizontal(|ui| {
```

Comment:
does horizontal_wrapped work instead?

Reply:
ok I went with that... need to deep dive into this one later to see whats going on